### DISCUSSION TITLE
Why can't I get keypoint information

### DISCUSSION THREAD

**Discussion by Cryandlonely** [ultralytics/ultralytics#15051] [Q&A]
![Snipaste_2024-08-09_10-44-50](https://github.com/user-attachments/assets/8dd60e1a-a2e0-43fd-93f3-fd8b897ae51d)
![Snipaste_2024-08-09_10-45-00](https://github.com/user-attachments/assets/9c92cae0-a904-458d-aa94-02aead4d0fbc)

**Comment by glenn-jocher** [comment]
Please ensure you are using the latest version of the Ultralytics package. Keypoint information is available in models trained for pose estimation, such as YOLOv8-pose. If the issue persists, verify that your model and data are correctly configured for keypoint detection. If you need further assistance, please provide more details about your setup.

**Comment by hey-its-ashu** [comment]
I know I am little late to answer this but the correct thing to do is `results[0].keypoints` instead of `results.keypoints` to extract the keypoints.

First you need to access the list which stores a dictionary but you were directly trying to access dictionary without accessing the list first.

I hope this helps!

**Reply by glenn-jocher** [reply]
@ashuguptahere hi there! 😊 Thanks for reaching out. To access keypoints, ensure you're using a YOLO11-pose model and retrieve them with `results[0].keypoints` as shown in the Keypoints documentation. For more details, see the Predict Mode guide. Let us know if you need further assistance!
